src/containers/edit-namespace/edit-namespace.tsx

```diff
@@ -212,6 +213,20 @@ class EditNamespace extends React.Component<RouteComponentProps, IState> {
     );
   }
 
+  private isSaveDisabled() {
+    const namespace = this.state.namespace;
+    let disabled = false;
+    namespace.links.forEach((link) => {
+      if (
+        NamespaceForm.validateName(link).validated == 'error' ||
+        NamespaceForm.validateUrl(link).validated == 'error'
+      ) {
+        disabled = true;
+      }
+    });
+    return disabled;
```

Comment:
```suggestion
    return namespace.links.some((link) =>
      NamespaceForm.validateName(link).validated == 'error' ||
      NamespaceForm.validateUrl(link).validated == 'error');
```

(+- prettier :))

[Array.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)